frontend/src/metabase/css/dashboard.module.css

```diff
@@ -6,6 +6,28 @@
 
 /* Night mode */
 .Dashboard.DashboardNight {
+  /**
+    * Apply dark mode colors to semantic color variables.
+    * Referenced from EmbedFrame.module.css
```

Comment:
It would be best to share a selector for dark mode that would combine `.DashboardNight` for dashboard and `[data-metabase-theme="night"][data-metabase-theme="night"]` for embedding, but let's do that later.

Reply:
If we have to change the theme logic, I think it would be best to use the theme provider that we use in the sdk, we shouldn't have too many ways of styling things, especially because the underlying themed components are the same